src/Parser/FrostMage/Modules/Items/Tier20_2set.js

```diff
@@ -2,26 +2,35 @@ import React from 'react';
 import SPELLS from 'common/SPELLS';
 import SpellIcon from 'common/SpellIcon';
 import SpellLink from 'common/SpellLink';
-import { formatPercentage } from 'common/format';
 import Combatants from 'Parser/Core/Modules/Combatants';
-
+import { formatNumber } from 'common/format';
 import Module from 'Parser/Core/Module';
+import getDamageBonus from '../MageCore/GetDamageBonus';
 
 
 class Tier20_2set extends Module {
   static dependencies = {
     combatants: Combatants,
   }
 
+  damage = 0;
+
+  on_initialized() {
+	this.active = this.combatants.selected.hasBuff(SPELLS.FROST_MAGE_T20_2SET_BONUS_BUFF.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if (this.combatants.selected.hasBuff(SPELLS.FROZEN_MASS.id)) {
+      this.damage += getDamageBonus(event, 0.2);
+    }
```

Comment:
As kfinch said, AFAIK Frozen Mass increases _critical damage_, not all damage, so this should be encased in one more `if (event.hitType === HIT_TYPES.Crit)` provided you import HIT_TYPES from Parser/Core/HIT_TYPES.

Reply:
The preferred way is to return early instead of encasing with if-statements.

Reply:
Also it's recommended to extract the `0.2` before class as a constant, perhaps "FROZEN_MASS_DAMAGE_BONUS" (no magic numbers).